config/webpack/environment.js

```diff
@@ -1,3 +1,11 @@
 const { environment } = require('@rails/webpacker')
 
+const webpack = require('webpack')
+environment.plugins.prepend('Provide',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/booking_tours_01/pullRequest?prid=7835277)
```suggestion
environment.plugins.prepend("Provide",
```